src/components/directory-content.js

```diff
@@ -673,6 +681,45 @@ const DirectoryContent = () => {
         );
     };
 
+    const sort = useCallback(
+        (datakey, isAsc, isNumeric) => {
+            const rowsCopy = [...rows].map((row, index) => [row, index]);
+            if (datakey === 'type') {
```

Comment:
must test if datakey is null first
see code in commons-ui there is occurences of :
```sortFromProps(null, false, false);```